### PR TITLE
Sfm plus add velocity hold (2)

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAttitudeHold.sqf [hunk 1]
```diff
@@ -28,7 +28,20 @@ if ( _heli getVariable "fza_ah64_attHoldActive" && !(_heli getVariable "fza_ah64
...
+    if (_subMode == "vel") then {
+        (_heli getVariable "fza_ah64_attHoldDesiredVel")
+            params ["_setVelX", "_setVelY"];
 
+        private _roll  = [_pidRoll,  _deltaTime, _setVelX, _curVelX] call fza_fnc_pidRun;
+        _roll          = [_roll,  -1.0, 1.0] call BIS_fnc_clamp;
+        private _pitch = [_pidPitch, _deltaTime, _setVelY, _curVelY] call fza_fnc_pidRun;
+        _pitch         = [_pitch, -1.0, 1.0] call BIS_fnc_clamp;
+
+        systemChat format ["Vel Hold! Des X %1 -- Des Y %2", _setVelX, _setVelY];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAttitudeHold.sqf:40]: Please remove this systemchat

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp [hunk 2]
```diff
@@ -37,6 +37,6 @@
 
 #define HDG_HOLD_SPEED_SWITCH_DECEL   15.433  //30kts GS
 #define HDG_HOLD_SPEED_SWITCH_ACCEL   20.577  //40kts GS
-#define HDG_HOLD_BREAKOUT_VALUE       0.03
-#define VEL_HOLD_BREAKOUT_VALUE       0.06
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp:41]: Please remove these changes

Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp:41]: Please remove these changes

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Great start, only a few small changes to be made

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAttitudeHold.sqf (hunk 1 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAttitudeHold.sqf:40]: Removed

**Review by mattysmith22 — APPROVED** [review]
Approved! Thanks. Ready for squash merge

**Pull request merged**